Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/server/BaseService.java

```diff
@@ -93,6 +93,21 @@ protected String getRemoteAddress(WebServiceContext context) {
     }
 
     /**
+     * Returns the local client host address
+     *
+     * @param context
+     * @return
+     */
+    protected String getLocalAddress(WebServiceContext context) {
+        String remoteAddress = null;
+        if (context != null && context.getMessageContext() != null && context.getMessageContext().get(AbstractHTTPDestination.HTTP_REQUEST) != null) {
```

Comment:
Be sure to wrap anywhere you extend beyond 120 characters (the formatter you imported will drop a red line down the screen after the 120th character.  GitHub also cuts off at 120 characters, and makes anything beyond difficult to review.